students/philip_behrend/lesson01/inventory_management/main.py

```diff
@@ -0,0 +1,80 @@
+"""Launches the user interface for the inventory management system"""
+import sys
+import inventory_management.market_prices as mkt
+from inventory_management.electric_appliance_class import ElectricAppliances
+from inventory_management.inventory_class import Inventory
+from inventory_management.furniture_class import Furniture
+
+def main_menu(user_prompt=None):
+    """Main menu options"""
+
+    valid_prompts = {"1": add_new_item,
+                     "2": item_info,
+                     "q": exit_program}
+    options = list(valid_prompts.keys())
+
+    while user_prompt not in valid_prompts:
+        options_str = ("{}" + (", {}") * (len(options)-1)).format(*options)
+        print("Please choose from the following options ({options_str}):")
+        print("1. Add a new item to the inventory")
+        print("2. Get item information")
+        print("q. Quit")
+        user_prompt = input(">")
```

Comment:
Add this line for debug purposes:

`print(f"USER PROMPT RECEIVED: {user_prompt}")`

This will provide another clue on what the issue is.